spec/spec_helper.rb

```diff
@@ -8,6 +8,26 @@
 require 'property_sets/delegator'
 require 'support/database'
 require 'support/account'
-require 'active_record/fixtures'
 
 I18n.enforce_available_locales = false
+
+# http://stackoverflow.com/questions/5490411/counting-the-number-of-queries-performed/43810063#43810063
+module QueryAssertions
+  def sql_queries(&block)
+    queries = []
+    counter = ->(*, payload) {
+      queries << payload.fetch(:sql) unless ["CACHE", "SCHEMA"].include?(payload.fetch(:name))
```

Comment:
`%w[CACHE SHEMA]`

Reply:
idk ... find the 'normal' form more readable for short lists ...